protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1128,8 +1232,9 @@ message IntervalToCandleId {
     string candle_id=2;
 }
 
-// A list of interval to candle id mappings for a given market
+// A list of mappings interval to candle ID for a given market
 message ListCandleIntervalsResponse {
+    // Mappings interval to candle ID
```

Comment:
I don't quite understand this one either, specifically "mappings interval"

Reply:
Same as the above comment explanation, but this is the filed that contains it